TRANSLATIONS.rst

```diff
@@ -0,0 +1,145 @@
+🐍 🌍 🚀 Translation Release Strategy
+======================================
+
+Thanks for your interest in translating the micro:bit Python Editor.
+Translations for all the Micro:bit Educational Foundation’s editors and
+apps are contributed by an amazing team of volunteers and we’d love for
+you to be a part of this too.
+
+This document sets out the strategy and process for how the Foundation
+manages the translation of the Python Editor hosted at
+python.microbit.org
+
+1. Translation and Management in the Python Editor
+--------------------------------------------------
+
+Translation is the act of taking the written materials in one language
+and converting them into another language in the most meaningful way
+possible. In terms of the micro:bit Python Editor, translation happens
+on the `strings marked for translation in the projects'
+codebase <https://github.com/bbcmicrobit/PythonEditor/lang/en.js>`__. The default language of
+the editor is English and you may select another available language
+using the Globe icon in the main menu.
```

Comment:
We don't really have a main menu in the Python Editor. Not sure if we actually have a name for the buttons area, maybe we can refer to it as just the "Globe button"? Do you think that would be clear enough?

Reply:
Yes or 'Globe button in the navigation'? or do we just not refer to that area at all?

Reply:
Navigation to me sounds the browser navigation bar. Does the support articles refer to this area with any specific terms?

Reply:
I've always called it the menu, so could we just remove 'main'